subsys/net/lib/sntp_sock/sntp_sock.c

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_MODULE_NAME net_sntp_sock
+#define NET_LOG_LEVEL CONFIG_SNTP_SOCK_LOG_LEVEL
+
+#include <net/sntp_sock.h>
+#include "sntp_sock_pkt.h"
+
+#define SNTP_LI_MAX 3
+#define SNTP_VERSION_NUMBER 3
+#define SNTP_MODE_CLIENT 3
+#define SNTP_MODE_SERVER 4
+#define SNTP_STRATUM_KOD 0 /* kiss-o'-death */
+#define OFFSET_1970_JAN_1 2208988800
+
+#define SNTP_CTX_SRV_SOCKADDR(ctx) (ctx->net_app_ctx.default_ctx->remote)
+
+static void sntp_pkt_dump(struct sntp_pkt *pkt)
+{
+	if (!pkt) {
+		return;
+	}
+
+	NET_DBG("li               %x", LVM_GET_LI(pkt->lvm));
+	NET_DBG("vn               %x", LVM_GET_VN(pkt->lvm));
+	NET_DBG("mode             %x", LVM_GET_MODE(pkt->lvm));
+	NET_DBG("stratum:         %x", pkt->stratum);
+	NET_DBG("poll:            %x", pkt->poll);
+	NET_DBG("precision:       %x", pkt->precision);
+	NET_DBG("root_delay:      %x", pkt->root_delay);
+	NET_DBG("root_dispersion: %x", pkt->root_dispersion);
+	NET_DBG("ref_id:          %x", pkt->ref_id);
+	NET_DBG("ref_tm_s:        %x", pkt->ref_tm_s);
+	NET_DBG("ref_tm_f:        %x", pkt->ref_tm_f);
+	NET_DBG("orig_tm_s:       %x", pkt->orig_tm_s);
+	NET_DBG("orig_tm_f:       %x", pkt->orig_tm_f);
+	NET_DBG("rx_tm_s:         %x", pkt->rx_tm_s);
+	NET_DBG("rx_tm_f:         %x", pkt->rx_tm_f);
+	NET_DBG("tx_tm_s:         %x", pkt->tx_tm_s);
+	NET_DBG("tx_tm_f:         %x", pkt->tx_tm_f);
+}
+
+static s32_t parse_response(u8_t *data, u16_t len, u32_t orig_ts,
+				u64_t *epoch_time)
```

Comment:
parameter alignement, it should be below last '(' + 1 space.

And return int, not s32_t